aikau/src/main/resources/alfresco/services/actions/CopyMoveService.js

```diff
@@ -124,10 +134,18 @@ define(["dojo/_base/declare",
                documents: documents,
                responseScope: payload.alfResponseScope
             };
+         var createLinkButtonLabel = payload.createLinkButtonLabel || "services.ActionService.createLink",
+             createLinkResponseTopic = this.generateUuid() + "_ALF_CREATE_LINK_LOCATION_PICKED",
+             createLinkPublishPayload = {
+                        nodes: nodes,
+                        documents: documents,
+                        responseScope: payload.alfResponseScope
+                     };
 
          var firstFileName = documents.length ? documents[0].fileName : null;
          var fileName = (nodes.length === 1 && firstFileName) ? firstFileName : this.message("services.ActionService.copyMoveTo.multipleFiles");
          this._actionHandle = this.alfSubscribe(responseTopic, lang.hitch(this, this.onCopyMoveLocationSelected, urlPrefix, payload.copy), true);
+         this._actionHandle = this.alfSubscribe(createLinkResponseTopic, lang.hitch(this, this.onCreateLinkLocationSelected), true);
          this.alfPublish("ALF_CREATE_DIALOG_REQUEST", {
```

Comment:
This isn't something I'd have expected you to have known... but since this function was originally written we've started using the `alfresco/core/topics` module for topic constants. So here we could use `topics.CREATE_DIALOG`. We'd also want to add a `@fires module:alfresco/core/topics#CREATE_DIALOG` statement to the JSDoc block...

...again, this isn't something I'd expect you to have known, but it would be useful if we can update modules as we go.